src/__forks__/fetch.js

```diff
@@ -11,5 +11,11 @@
 
 'use strict';
 
-require('whatwg-fetch');
-module.exports = self.fetch.bind(self);
+// This hopefully supports the React Native case, which is already bringing along
+// its own fetch polyfill. That should exist on `global`. If that doesn't exist
+// then we'll try to polyfill, which might not work correctly in all environments.
+if (global.fetch) {
+  module.exports = global.fetch.bind(global);
+} else {
+  module.exports = require('isomorphic-fetch');
+}
```

Comment:
I never got around to revisiting the feedback about this in #61. If it works then great but I just wanted to make sure it was all good. Sounds like combined with the providesModule bits it might have fixed the issue.